packages/react/src/components/ComposedModal/ComposedModal.js

```diff
@@ -127,6 +127,10 @@ export default class ComposedModal extends Component {
     } else if (prevProps.open && !this.props.open) {
       this.beingOpen = false;
     }
+    document.body.classList.toggle(
```

Comment:
Do we need to worry about the second argument to toggle not being supported in older versions of IE?

Reply:
Good point - Thought that we have the polyfill, but saw it's not the case for the React repo. Added it with tests.